src/services/codefixes/returnValueCorrect.ts

```diff
@@ -0,0 +1,211 @@
+/* @internal */
+namespace ts.codefix {
+    const fixId = "returnValueCorrect";
+    const fixIdAddReturnStatement = "fixAddReturnStatement";
+    const fixIdRemoveBlockBodyBrace = "fixRemoveBlockBodyBrace";
+    const fixIdWrapTheBlockWithParen = "fixWrapTheBlockWithParen";
+    const errorCodes = [
+        Diagnostics.A_function_whose_declared_type_is_neither_void_nor_any_must_return_a_value.code,
+        Diagnostics.Type_0_is_not_assignable_to_type_1.code,
+        Diagnostics.Argument_of_type_0_is_not_assignable_to_parameter_of_type_1.code
+    ];
+
+    enum FixKind {
+        MissingReturnStatement,

```

Comment:
Arguably, these are problem kinds.  The corresponding fix kinds would be "AddReturnStatement" and "AddParentheses".

Reply:
Why are there only two kinds of fixes?  Shouldn't there also be one for removing the braces?

Reply:
And why not just use the fix IDs from above?